DGWG-Charter.md

```diff
@@ -0,0 +1,140 @@
+* Ballot for approval of Draft Guideline as a Final Guideline
+* Final Guideline for WG publication
+
+#### 2.2 **Other Deliverables** 
+
+No other deliverables are expected of the Definitions and Glossary Working Group.
+
+#### 2.3 **Milestones** 
+
+Milestones may be modified or replaced by consensus of the Working Group, without requiring modification of this Charter. The initial milestones for the Draft Guideline are as follows:
+
+1. Identify terms defined in documents published by other Chartered Working Groups within the Forum
+2. Consolidate all terms along with all differences in their various definitions
+3. Identify similar terms and consolidate where warranted
+4. Establish definitions for each identified term, bearing in mind the potential variety of use-cases
+    1. Separate defined terms as needed in order to provide clarity around use within different use-cases
+5. Finalize defined terms in a Draft Guideline and share within the Forum for discussion and review
+
+### **3. Membership**
+
+#### **3.1 Membership Criteria and Application Process**
+
+This Working Group shall have the following categories of membership: Certificate Consumer, Certificate Issuer, Associate Member, and Interested Party.
+
+Applicants must be members of the CSCWG, the SMCWG, or the SCWG, and may join the Definitions and Glossary Working Group only with such status or class as they hold in such other Chartered Working Groups.
+
+In accordance with the IPR Policy, Members that choose to participate in the Definitions and Glossary Working Group must declare their participation and class of membership prior to participating. The Chair of the Definitions and Glossary Working Group shall establish a mailing list for use by the Working Group, including for declarations of participation, and manage it in accordance with the Bylaws, the IPR Policy, and the IPR Agreement. A Member must declare its participation in the Definitions and Glossary Working Group on the Working Group’s mailing list.
+
+The Definitions and Glossary Working Group shall include Interested Parties and Associate Members as defined in the Bylaws.
+
+#### **3.2 Voting Structure**
+
+The Working Group shall consist of two classes of voting members: Certificate Issuers and Certificate Consumers 
+
+In order for a ballot to be adopted by the Definitions and Glossary Working Group, two-thirds or more of the votes cast by the Certificate Issuers must be in favor of the ballot and more than half of the votes cast by the Certificate Consumers must be in favor of the ballot. At least one member of each class must vote in favor of a ballot for it to be adopted. Quorum is the average number of Member organizations (cumulative, regardless of Class) that have participated in the previous three Definitions and Glossary Working Group Meetings or Teleconferences (not counting subcommittee meetings thereof). For transition purposes, if three meetings have not yet occurred, quorum is ten (10).
```

Comment:
@clintwilson Sorry, I should have spotted this during my earlier review, but this language has the same issue as the S/MIME charter, in that it would be impossible to reach quorum if the charter rules are used: If quorum is the average number of members participating, regardless of class, that means non-voting members are also counted, meaning you could never have enough votes.

Since all charters need to be updated, we might as well do this one correctly right away. 

Borrowing language from the ByLaws, which divides the average in half +1, and also setting the calculation to only count voting members, I'd like to propose:

```suggestion
In order for a ballot to be adopted by the Definitions and Glossary Working Group, two-thirds or more of the votes cast by the Certificate Issuers must be in favor of the ballot and more than half of the votes cast by the Certificate Consumers must be in favor of the ballot. At least one member of each class must vote in favor of a ballot for it to be adopted. Quorum is defined as more than half of the average number of Voting Member organizations (cumulative) that have participated in the previous three Definitions and Glossary Working Group Meetings or Teleconferences (not counting subcommittee meetings thereof). For transition purposes, if three meetings have not yet occurred, quorum is ten (10).
```

This puts quorum at the average of voting members over the last 3 meetings, taking half of that, and adding one (more than half of the average), and follows the Bylaws calculation